src/LibraryViewExtensionWebView2/Handlers/IconResourceProvider.cs

```diff
@@ -225,18 +225,9 @@ private string GetIconAsBase64(IconUrl icon, out string extension)
             var assemblyName = String.Join("", temp.Take(temp.Length - 1));
             var rm = new ResourceManager(assemblyName + imagesSuffix, assembly);
 
-            using (var image = (Bitmap)rm.GetObject(icon.Name))
-            {
-                if (image == null) return null;
-
-                var tempstream = new MemoryStream();
-
-                image.Save(tempstream, image.RawFormat);
-                byte[] imageBytes = tempstream.ToArray();
-                tempstream.Dispose();
-                string base64String = Convert.ToBase64String(imageBytes);
-                return base64String;
-            }
+            byte[] imageBytes = rm.GetObject(icon.Name) as byte[];
+            string base64String = Convert.ToBase64String(imageBytes);
+            return base64String;
```

Comment:
The change here is actually an optimization.  Currently the exception for casting to Bitmap is caught outside of this function.  Maybe we also do something here?